libraries/Microsoft.Bot.Builder.Dialogs/DialogContext.cs

```diff
@@ -659,7 +659,7 @@ public async Task<bool> EmitEventAsync(string name, object value = null, bool bu
         /// <returns>A string representing the current locale.</returns>
         public string GetLocale()
         {
-            return Context.TurnState.Get<string>("turn.locale");
+            return (Context as TurnContext).Locale;
```

Comment:
This appears to be reverting a seemingly intentional change @carlosscastro made here: https://github.com/microsoft/botbuilder-dotnet/pull/4860

Reply:
Right, I recalled this change in previous discussion. Due to the path is changed, I did some modification this time. But still follows Carlos' suggestion.

Reply:
Good deal, just wanted to double check.